app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class VotesController < ApplicationController
+  def new
+  end
+
+  def create
+    @vote = Vote.new(vote_params)
+    @vote.save
+    redirect_to works_path
```

Comment:
You should be checking the result of `save` here, and reporting it (success or failure) to the user so they know what happened.